app/javascript/react/components/AnimalReviewForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from "react";
+import OwlIcon from "./OwlIcon";
+
+const AnimalReviewForm = (props) => {
+  const [animalReview, setAnimalReview] = useState({
+    title: "",
+    description: "",
+    rating: 0,
+  });
+
+  const clearForm = (event) => {
+    event.preventDefault();
+    setAnimalReview({
+      title: "",
+      description: "",
+      rating: 0,
+    });
+  };
+
+  const handleInputChange = (event) => {
+    setAnimalReview({
+      ...animalReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+    });
+  };
+
+  const onSubmitHandler = (event) => {
+    event.preventDefault();
+    props.submittedHandler(animalReview);
+  };
+
+  let owlIcons = [];
+  for (let i = 1; i <= 5; i++) {
+    const handleSetRating = () => {
+      setAnimalReview({
+        ...animalReview,
+        ["rating"]: i,
+      });
+    };
+
+    owlIcons.push(
+      <OwlIcon
+        id={i}
+        keyNumber={i}
+        handleSetRating={handleSetRating}
+        className={i <= animalReview.rating ? "head selected" : "head"}
+      />
+    );
+  }
+
```

Comment:
I don't see a ratings setting in your review form? How will they know how many owls will be made?